.github/workflows/release.yaml

```diff
@@ -54,6 +54,7 @@ jobs:
 
   gh-release:
     if: github.ref_type == 'tag'
+    needs: publish
```

Comment:
This will prevent a Github release if publishing on crates-io was unsuccessful.

Reply:
Cool!